README.md

```diff
@@ -262,24 +220,226 @@ We can also pull data from different forms and join the various form data. Here
+  "STYLES": [
+    {
+      "label": "Light",
+      "style": "mapbox://styles/mapbox/light-v9"
+    },
+    {
+      "label": "Streets",
+      "style": "mapbox://styles/mapbox/streets-v9"
+    },
+    {
+      "label": "Satellite",
+      "style": "mapbox://styles/mapbox/satellite-v9"
+    }
+  ],
+  "LAYERS": [
+    "ken-health-sites",
+  ]
+}
+```
+
+Let's go over each of the keys in the `site-config.json` file
+
+1. `APP` **(Required)** : The `APP` key expects and object and ontains properties for the site settings. The following are the properties supported
+
+- `mapConfig` **(Required)** : An object that contains keys for setting the values for the properties for
+  interacting with the map. More information about each of these properties including their defaults
+  can be found at the [Mapbox GL JS API Reference](https://docs.mapbox.com/mapbox-gl-js/api/map/)
+
+  - `container` **(Required)** : Your HTML element in which Mapbox GL JS will render the map
+  - `style` **(Required)** : The default Mapbox style(basemap) to be loaded from the Mapbox API of the
+    form `mapbox://styles/:owner/:style`
+  - `center` **(Required)** : The initial geographical centerpoint of the map (Uses longitude, latitude coordinated order).
+  - `zoom` **(Required)** : The initial zoom level of the map.
+  - `mapBounds` **(Optional)** : Map points when exporting i.e from south west to north east.
+      e.g 
+
+      ```
+      {
+        "_sw": {
+          "lng": 22.13636132811544,
+          "lat": -0.14320447808067627
+        },
+        "_ne": {
+          "lng": 62.89563867185916,
+          "lat": 11.188804478371793
+        }
+      }
+      ```
+  - `preserveDrawingBuffer` : whether to preserve the buffers until manually cleared or overwritten.
+
+- `accessToken` **(Required)** :  Your Mapbox basemap style API access token.
+- `hyperLink` **(Optional)** : Add links and description to menu items. 
+- `authConfigApiMap` **(Optional)** : Supplemental auth api string, can either be data or metadata. Defaults to metadata 
+    e.g https://api.ona.io/api/v1/data/467840.csv or https://api.ona.io/api/v1/metadata/467840.csv
+- `apiAccessToken` **(Optional)** : Ona data/external api access token.
+- `includeNavControls` **(Optional)** : Boolean value determines if map controls will be included. Defaults to true
+- `removeMapScale` **(Optional)** : Removes map scale. Defaults to false
+- `showSinglePeriods` **(Optional)** : Show timeseries for single periods. Defaults to false
+- `NULL_LAYER_TEXT` **(Optional)** : Custom Text to be shown when user has no access to a layer. Defaults to ``` You don't have permision to view this category```
+- `appIcon` **(Optional)** : The relative path to image that will be displayed as the logo on the top of the title bar.
+- `appName` **(Optional)** : The name for the app that will be displayed on the title bar.
+- `appColor` **(Required)** : The app primary color that will be displyed most frequently across the screens such
```

Comment:
> `appColor` **(Required)** : The app primary color that will be displyed most frequently across the screens such

Let's have all required before the optional
**displyed**  Let's correct the typo